### PR TITLE
Add johannhj cadott wisconsin file

### LINKED ISSUE #6: johannhj cadott wisconsin
- [ ] Create a **branch**
- [ ] Add file/**commits**
- [ ] Open a **PR**
- [ ] Have a **discussion** about the PR (collaborate)
- [ ] Make **edits**
- [ ] Get :ship: 
- [ ] **Merge** PR

### PR DESCRIPTION
**Pull request by johannhj**
closes #6 

hi @ez4gis and @tingleeGIS, does this look okay?

### CODE REVIEW

File: johannhj-cadottwisconsin.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#Cadott, WI
+
+###Famous For
+-Indian statue Jean Baptiste Cadotte
+
+###Interesting Facts
+--45th parallel "Half-way between the Equator & North Pole"
```
Comment (ghost) [inline johannhj-cadottwisconsin.md:7]: :+1: never been there but sounds really :cool: ! :smiley:

**Comment by johannhj** [comment]
@tingleeGIS - Would you say this file is ready to go? I should add a photo like you..........I would if I knew how!

**Comment by johannhj** [comment]
yellow river

**Comment by johannhj** [comment]
indian man

**Pull request merged**